### ISSUE TITLE
operator_usage_whitespace does not work for the ternary operator

### ISSUE THREAD

**Issue by yonat-ateam** [realm/SwiftLint#3965]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

The `operator_usage_whitespace` rule fails to identify missing spaces around the ternary operator (`? :`).

### Environment

* SwiftLint version: 0.47.1
* Installation method : cloned this repo and added tests
* Paste your configuration file:

```yml
opt_in_rules:
  - operator_usage_whitespace
```

* Do you have a sample that shows the issue?

Example that should trigger the rule:

```swift
  let foo = bar ? 0:1
```

Expected fix:

```swift
  let foo = bar ? 0 : 1
```

**Comment by marcelofabri** [comment]
Fixed in #3962
